phpcs.xml.dist

```diff
@@ -103,4 +104,19 @@
 	<rule ref="PEAR.NamingConventions.ValidClassName.Invalid">
 		<exclude-pattern>/phpunit/*</exclude-pattern>
 	</rule>
+
+	<rule ref="Gutenberg.CodeAnalysis.GuardedFunctionAndClassNames">
+		<exclude-pattern>./phpunit/*</exclude-pattern>
+		<exclude-pattern>./packages/*</exclude-pattern>
+		<exclude-pattern>./bin/generate-gutenberg-php</exclude-pattern>
+		<properties>
+			<property name="functionsWhiteList" type="array">
+				<element value="/^_?gutenberg.+/"/>
+			</property>
+			<property name="classesWhiteList" type="array">
+				<element value="/^Gutenberg.+/"/>
+				<element value="/^WP_.+_Gutenberg$/"/>
```

Comment:
The question is: should classes like `WP_Some_Class_Name_Gutenberg` be allowed? I suppose it's quite unlikely that the `_Gutenberg` suffix won't be removed during backporting to Core, so I would say it's fine to whitelist such classes.